src/lib/util/state.ts

```diff
@@ -103,11 +103,11 @@ export const updateConfig = (config: string, updateEditor: boolean): void => {
 	});
 };
 
-export const toggleDarkTheme = (dark: boolean): void => {
+export const toggleDarkTheme = (dark: boolean, theme: string): void => {
```

Comment:
```suggestion
export const setTheme = (theme: 'dark' | 'default' | 'forest'): void => {
```

There's no need to pass a boolean along with theme name.